packages/scanner/zxing/android/src/main/kotlin/org/openfoodfacts/scanner_zxing/ZXingUtils.kt

```diff
@@ -0,0 +1,62 @@
+package org.openfoodfacts.scanner_zxing
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.graphics.Matrix
+import android.util.Log
+import com.google.zxing.BinaryBitmap
+import com.google.zxing.MultiFormatReader
+import com.google.zxing.NotFoundException
+import com.google.zxing.RGBLuminanceSource
+import com.google.zxing.common.HybridBinarizer
+
+
+object ZXingUtils {
+
+    fun extractBarcodeFromImage(path: String?, orientation: Int?): String? {
```

Comment:
Same comment as before

Reply:
If you have a look at the content of this method, I return a null string in the case of null args